builder/FindGlobals.cmake

```diff
@@ -79,20 +78,20 @@ if( Linux OR Darwin )
     set( git_commit "" )
     git_describe( git_commit )
 
-    add_definitions( -DMSDK_BUILD=\"$ENV{BUILD_NUMBER}\")
+    set( common_flags "${common_flags} -DMSDK_BUILD=\\\"$ENV{BUILD_NUMBER}\\\"" )
```

Comment:
why so many slashes? :)

Reply:
Because we need to get `-DMSDK_BUILD="\"<string>"\"`